app/javascript/src/Plans/index.js

```diff
@@ -3,9 +3,9 @@
 export * from './components/DefaultPlanSelect'
 export * from './components/DefaultPlanSelectCard'
 export * from './components/DefaultPlanSelectWrapper'
-// $FlowIgnore[cannot-resolve-module] name-mapper is replacing 'Plans'
+// $FlowFixMe[cannot-resolve-module] name-mapper is replacing 'Plans', to fix this rename module or fix name_mapper expression in flowconfig
```

Comment:
Do you know why `Plans` is replaced?

Reply:
Yes, our `name-mapper` setup replaces any term within the regexp (see .flowconfig) with the absolute path of porta. That's how we are able to use import paths like this. Example:
```jsx
import * from 'Plans/foo'
// is replaced during compilation with
import * from '/user/path/to/porta/src/javascript/src/Plans/foo'
```

Therefore anything that matches the same regexp will be replaced, for instance `ApplicationPlansTableCard`:
```jsx
export * from './components/ApplicationPlansTableCard'
// is replaced during compilation with
import * from './components/Application/user/path/to/porta/src/javascript/src/Plans/TableCard'
```

Reply:
This is why I removed the module app/javascript/src/Form in this PR, because any component named `SomethingForm.jsx` was being replaced and it's a PITA.